src/js/features/volume.js

```diff
@@ -78,7 +78,7 @@ Object.assign(MediaElementPlayer.prototype, {
 		mute.innerHTML = mode === 'horizontal' ?
 			`<button type="button" aria-controls="${t.id}" title="${muteText}" aria-label="${muteText}" tabindex="0"></button>` :
 			`<button type="button" aria-controls="${t.id}" title="${muteText}" aria-label="${muteText}" tabindex="0"></button>` +
-			`<a href="javascript:void(0);" class="${t.options.classPrefix}volume-slider" ` +
+			`<a href="about:blank" class="${t.options.classPrefix}volume-slider" ` +
```

Comment:
`#` is usually used here which is better than `about:blank` if something fails.

Reply:
Resolved.